src/style.css

```diff
@@ -1 +1,90 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+body {
+  box-sizing: border-box;
+
+  --text-color: #3d4e61;
+  --main-color: #e1e7ed;
+}
+
+.form-search-bar {
+  display: flex;
+  flex-direction: row;
+  margin: 20px 0 0;
+}
+
+.form-search-bar--big {
+  height: 70px;
+}
+
+.form-search-bar--small {
+  height: 42px;
+}
+
+.input-bar {
+  color: var(--text-color);
+  background-color: #fff;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  flex-direction: row;
+  width: 100%;
+  display: flex;
```

Comment:
The 'display: flex;' property is already defined in the '.form-search-bar' class, which is a parent of '.input-bar'. Consider removing this line to avoid redundancy.